app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from os import path
+import sys
+
+
+def copy_file(command: str) -> str or None:
+    cmd, main_file, new_file = command.split()
+    source = path.join(sys.path[0], main_file)
+    destination = path.join(sys.path[0], new_file)
+    try:
+        if cmd != "cp":
+            raise ValueError("Wrong command! You should use `cp`.")
+        if main_file == new_file:
+            raise ValueError("Cannot copy file to itself!")
+
+    # we can use copy() from shutil, but I've made it in more "vanilla" way
```

Comment:
If you have to leave a comment for your mentor, use your pull request comments, rather than code comments. Please, remove this comment block :)